lib/View/bookmarks_screen.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/View/utils/custom_bottom_nav_bar.dart';
+import 'package:gocast_mobile/View/utils/video_card_view.dart';
+class BookmarksScreen extends ConsumerWidget {
+  const BookmarksScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Bookmarks'),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.search),
+            onPressed: () {
+              // Implement search functionality
+            },
+          ),
+          IconButton(
+            icon: const Icon(Icons.filter_list),
+            onPressed: () {
+              // Implement more options functionality
+            },
+          ),
+        ],
+      ),
+      body: ListView(
```

Comment:
Refactor to single component (e.g.,`CourseList`) that you then use in `bookmarks_screen` and `download_screen` to avoid duplicate code.

Reply:
I think in bookmarks_Screen (pinnedcourses) it is a course, and in downloads its a video right?